app/views/notes/_note.html.slim

```diff
@@ -14,9 +14,8 @@
           =svg_symbol '#icon-remove-sign', class: 'icon', title: t('.delete')
     =time_tag note.created_at
       =link_to note.user.display_name, user_profile_path(:user_id => note.user.id)
-      =" "
+      |&nbsp;
       =time_ago_in_words(note.created_at)
-      =" ago"
```

Comment:
We seem to have lost the "ago".  Can we add it back in, please?  Translated, if possible.  (Or maybe it's blank in other languages, but it only makes sense in with the "ago" in English.)